CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+# Contributing Guide
+
+* [New Contributor Guide](#contributing-guide)
+  * [Ways to Contribute](#ways-to-contribute)
+  * [Find an Issue](#find-an-issue)
+  * [Ask for Help](#ask-for-help)
+  * [Pull Request Lifecycle](#pull-request-lifecycle)
+  * [Development Environment Setup](#development-environment-setup)
+    * [Requirements](#requirements)
+    * [Build](#build)
+    * [Install](#install)
+    * [Test](#test)
+  * [Pull Request Checklist](#pull-request-checklist)
+
+Welcome! We are glad that you want to contribute to our project!
+
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with including:
+
+* Problems found during a new developer environment setup.
+* Gaps in our Quickstart Guide or documentation.
+* Bugs in our automation scripts.
+
+If anything doesn't make sense, or doesn't work when you run it, please open a
+[issue](https://github.com/rpm-software-management/dnf5/issues/new) and let us know!
+If you file a bug on Fedora you could open a bug on
+[Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=dnf5).
```

Comment:
```suggestion
If you are experiencing problems with our project, let us know by opening an 
[issue](https://github.com/rpm-software-management/dnf5/issues/new) and let us know!
If you want to file a bug on Fedora, you can open the bug in
[Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=dnf5).
```

Not sure if all my suggestions make sense here, though :-)  Other option is to replace "with our project" with "the DNF5 package manager".